### ISSUE TITLE
Regarding a question raised in the readme

### ISSUE THREAD

**Issue by joe660** [AlexeyAB/darknet#7071]
![image](https://user-images.githubusercontent.com/74225912/100977927-a7101d00-357c-11eb-92db-4bffb728172e.png)

You said that we can use negative images to participate in training, what will the model learn about negative images? Confidence loss or what,  Thank you

**Comment by AlexeyAB** [comment]
It will learn `objectness` - this is the one part of Confidence.

**Comment by joe660** [comment]
> It will learn `objectness` - this is the one part of Confidence.



> It will learn `objectness` - this is the one part of Confidence.

Thank you very much for your reply. The negative images are all negative samples after prediction, so the confidence of the suspected negative sample prediction we want to learn will be high. Do you think this can be done? Take the top K confidences in the negative image and put them in the network for training. Make the network pay more attention to suspected areas. thank you very much

**Comment by AlexeyAB** [comment]
I don't understand, can you explain?

**Comment by joe660** [comment]
> I don't understand, can you explain?
Hello, I have only one target, but there will be suspected targets (false positives)。。
When adding negative images to the training set
However, the negative image does not contain any annotation, which means
all anchors are negative including those anchors contain suspected targets. It is
possible to optimize the loss functions of all anchors, but this will bias towards
normal regions since they are dominate. Those suspected target region still likely
to be recognized as positive since they don’t have enough chance to be trained.
Take the top K confidences in the negative image and put them in the network for training. Make the network pay more attention to suspected areas.

**Comment by joe660** [comment]
> I don't understand, can you explain?



> I don't understand, can you explain?

Hi, can you see my reply? Thank you

Hello, I have only one target, but there will be suspected targets (false positives)。。
When adding negative images to the training set
However, the negative image does not contain any annotation, which means
all anchors are negative including those anchors contain suspected targets. It is
possible to optimize the loss functions of all anchors, but this will bias towards
normal regions since they are dominate. Those suspected target region still likely
to be recognized as positive since they don’t have enough chance to be trained.
Take the top K confidences in the negative image and put them in the network for training. Make the network pay more attention to suspected areas. Is this feasible? Thank you

**Comment by AlexeyAB** [comment]
> However, the negative image does not contain any annotation, which means
> all anchors are negative including those anchors contain suspected targets. 

No. Negative samples must not contain required objects.

**Comment by joe660** [comment]
> No. Negative samples must not contain required objects.



> > However, the negative image does not contain any annotation, which means
> > all anchors are negative including those anchors contain suspected targets.
> 
> No. Negative samples must not contain required objects.

It does not contain a positive object, it is a false positive, it is not true, it just looks like the target object

**Comment by joe660** [comment]
> > However, the negative image does not contain any annotation, which means
> > all anchors are negative including those anchors contain suspected targets.
> 
> No. Negative samples must not contain required objects.

It does not contain a positive object and  it is a false positive, it just looks like the target object, Do you think this is ok thanks
